src/build-stack-trace.ts

```diff
@@ -25,7 +25,7 @@ const SRC_ROOT = __dirname;
  * @returns {String} - A string representation of the stack trace at the point
  *   where this method was invoked.
  */
-export function buildStackTrace(message) {
+export function buildStackTrace(message: string) {
```

Comment:
Should this change come with an update to the tsconfig.json?  Can we turn on noImplicityAny?  Or just use the default gts configuration?

Reply:
I'm working though the changes step-by-step.  The current code has `noImplicitAny` as `false` in tsconfig so that this PR can only focus on types.  The next PR will re-enable the standard gts configuration, and the PR after that will re-enable `gts fix`.

Reply:
👍